src/DateTimePicker/DateTimePicker.tsx

```diff
@@ -63,6 +63,8 @@ export interface DateTimePickerProps {
 	portalPopupTo?: React.ReactNode;
 	/** Show datepicker inline */
 	inline?: boolean;
+	/** Placement of datetime picker popup */
+	popperPlacement?: "top-start" | "top-end" | "bottom-start" | "bottom-end" | "left-start" | "left-end" | "right-start" | "right-end";
```

Comment:
Copying the values from another type is not good and can lead to all kinds de-sync issues. Instead use the power typing system in TS. For example, see how this is done in popup component:
```
import { Options } from "@popperjs/core";

position: Options["placement"];
```